composer.lock

```diff
@@ -338,26 +344,77 @@
             "time": "2015-11-12 16:18:56"
         },
         {
+            "name": "symfony/config",
+            "version": "v3.0.0",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/symfony/config.git",
+                "reference": "40bae8658dbbb500ebc19aa9fde22dc4295fc290"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/symfony/config/zipball/40bae8658dbbb500ebc19aa9fde22dc4295fc290",
+                "reference": "40bae8658dbbb500ebc19aa9fde22dc4295fc290",
+                "shasum": ""
+            },
+            "require": {
+                "php": ">=5.5.9",
```

Comment:
I don think we upped our requirements to 5.5 yet. I'm fine doing so for Drush9. I'd be OK with this patch being our starting point for Drush9. No strong opinion.